lib/verbalize/result.rb

```diff
@@ -18,7 +18,7 @@ def failed?
     alias_method :failure?, :failed?
 
     def to_ary
-      [outcome, value]
+      [outcome, @value]
```

Comment:
Since `value` will be going away from the base class as an accessor in v2.0, this needs to access the `@var` directly.